### PR TITLE
Improved handling for long words (#155)

### PR DESCRIPTION
**Pull request by vanwykjd**
Associated Issue: <https://github.com/ollelauribostrom/rebus/issues/155>

### Summary of Changes

- Width of character input-columns responds to varying word lengths

### CODE REVIEW

**Comment by vanwykjd** [comment]
@ollelauribostrom Thanks for looking into my changes and I appreciate your feedback! This isn't the first time I've been guilty of over-analyzing a situation...  

As you mentioned, I agree that some simple CSS changes should do the trick, and will look into making those changes tomorrow.

**Comment by vanwykjd** [comment]
@ollelauribostrom I had a feeling the added breakpoints may have been a little much... I tend to get caught up it in the details once I've envisioned something. 

I changed back to original the media queries and made the adjustments as suggested, but the appearance of the long words revert back to the original issue:

screen shot 2018-11-28 at 1 22 14 pm

My question is whether or not the improvements requested for this issue (https://github.com/ollelauribostrom/rebus/issues/155) regards all devices or just the large, and have the improvements be addressed for the mobile devices in issue https://github.com/ollelauribostrom/rebus/issues/152?

**Comment by ollelauribostrom** [comment]
I think for #155 we can just focus on making this look nice on desktops and keep the original style for mobile devices. Not sure on how the best way would be to solve this on mobile devices but we can look at that in #152 🙂

File: src/css/main.css [hunk 1]
```diff
@@ -88,13 +88,16 @@ body,
...
-  display: grid;
+  display: flex;
   justify-content: center;
-  grid-template-columns: repeat(auto-fit, 50px);
+  align-items: center;
 }
 
 .word__char {
   height: 50px;
+  width: 100%;
```
Comment (ollelauribostrom) [inline src/css/main.css]: You can just use `width: 50px` (no need for the max-width property as far as I understand 🙂 )

File: src/css/main.css [hunk 2]
```diff
@@ -88,13 +88,16 @@ body,
...
+  max-width: 50px;
+  margin: auto 0;
```
Comment (ollelauribostrom) [inline src/css/main.css]: No need for this margin property

File: src/css/main.css [hunk 3]
```diff
@@ -162,7 +165,6 @@ body,
   grid-column-start: 1;
   grid-column-end: 6;
   border: none;
-  background: #9c0629;
```
Comment (ollelauribostrom) [inline src/css/main.css]: This should not be removed since it fixes the progress-bar appearance in FF

**Pull request merged**